distributed/tests/test_worker.py

```diff
@@ -1180,33 +1180,39 @@ async def test_statistical_profiling_2(c, s, a, b):
 
 
 @gen_cluster(
-    nthreads=[("127.0.0.1", 1)],
+    nthreads=[("", 1)],
     client=True,
-    worker_kwargs={"memory_monitor_interval": 10},
+    config={
+        "distributed.worker.memory.target": False,
+        "distributed.worker.memory.spill": 0.7,
+    },
```

Comment:
no change from default but I like to spell out explicitly relevant settings - also in the case they change in the future